laba5.py

```diff
@@ -0,0 +1,130 @@
+"""
+Var 17
+"""
+from enum import Enum
+import math
+
+
+class Color(Enum):
+    """
+    Define a color
+    """
+    RED = 1
+    GREEN = 2
+    BLUE = 3
+
+
+class Point:
+    """
+    Define a point
+    """
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def __repr__(self):
+        return f"Point({self.x}, {self.y})"
+    def get_x(self):
+        """Get x"""
+        return self.x
+    def get_y(self):
+        """Get y"""
+        return self.y
+    def print_point(self):
+        """Print out point data"""
+        print(f"Point x = {self.x}, y = {self.y}")
+    def __del__(self):
+        """Destructor"""
+        print("Point has been deleted")
+
+
+class Polynom:
+    """
+    Define a polynom with color and points
+    """
+    def __init__(self, color, *points):
+        self.color = color
+        self.points = list(points)
+
+    def __repr__(self):
+        return f"Polynom({self.color.name}, {', '.join(map(str, self.points))})"
+    def get_points(self):
+        """Get list of points"""
+        return self.points
+
+    def get_color(self):
+        """Get color of a point"""
+        return self.color
+    def print_polynom(self):
```

Comment:
add empty line to split method boundaries
```suggestion
        return self.color
     
    def print_polynom(self):
```